### PR TITLE
Closed New York Life

### PR DESCRIPTION
**Pull request by hemanthy1**
There's a bug saying the snap closing hasnt been merged yet and if you click on files changed, those have not changed

### CODE REVIEW

**Comment by hemanthy1** [comment]
Fixes #2125

File: README.md [hunk 1]
```diff
@@ -145,8 +145,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | UPS | Atlanta, GA | **🔒 Closed 🔒** Software Engineer Intern |
 | Snap Inc. | Los Angeles; Palo Alto; Seattle |  **🔒 Closed 🔒** Software Engineering Intern, Summer 2024 |
 | GE Aerospace | Multiple US Locations | [Digital Technology Intern](https://ge.wd5.myworkdayjobs.com/en-US/GE_ExternalSite/job/GE-Aerospace-US-Digital-Technology-Intern---Summer-2024_R3718425-1) (No Sponsorship) <br/> [Engineering Engines Computer or Software Engineering Intern](https://ge.wd5.myworkdayjobs.com/en-US/GE_ExternalSite/job/GE-Aerospace-Engineering-Engines-Computer-or-Software-Engineering-Intern---Summer-2024_R3726060) (No Sponsorship) <br/> [Systems Engineering Intern](https://ge.wd5.myworkdayjobs.com/en-US/GE_ExternalSite/job/GE-Aerospace-Systems-Engineering-Intern---Summer-2024_R3696296-1) (No Sponsorship) |
-| New York Life | New York, NY | [Technology Summer Intern](https://jobs.newyorklife.com/job/New-York-2024-Technology-Summer-Internship-Program-NY-10001/1055772100/) |
-| [Johnson & Johnson](https://jobs.jnj.com/en/jobs/2306128806w/technology-internship-2024-summer-jjt-intern/) | Multiple Locations | Technology Summer Intern (No Sponsorship) |
```
Comment (tianyizheng02) [inline README.md:149]: Why delete J&J?

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hemanthy1) [reply README.md:149]: Completely accident sorry! added back now

Comment (hemanthy1) [reply README.md:149]: Also added new GM position

**Comment by tianyizheng02** [comment]
@hemanthy1 Please make a separate branch for each of your changes. They're not related to each other and shouldn't all get lumped into this PR.

**Comment by hemanthy1** [comment]
> @hemanthy1 Please make a separate branch for each of your changes. They're not related to each other and shouldn't all get lumped into this PR.

Would you just prefer I close this and open up two seperate PRs?

**Comment by tianyizheng02** [comment]
Yeah, I'd appreciate it if you could do that

**Pull request closed without merging**